src/Plugin/jsonapi/FieldEnhancer/CardLinkEnhancer.php

```diff
@@ -132,6 +134,17 @@ public function getCardFields(EntityInterface $node) {
         $paragraph = $paragraph_id ? Paragraph::load($paragraph_id) : '';
         if ($paragraph && $paragraph->field_paragraph_date_range->getValue()) {
           $event_date = $paragraph->field_paragraph_date_range->getValue()[0];
+          // Parse date with GMT timezone.
+          $storageTz = DateTimeItemInterface::STORAGE_TIMEZONE;
```

Comment:
Please be consistent with the name conventions of variables if you are going to use camel case or snake case.

Reply:
Code is formatted now with snake_case